# Reference Data

We keep a number of reference data sets available on Katana at `/data/` so that we don't accidentally - for instance - end up with 150 copies of the Human Genome in user's home directories.

As these are reference data, they don't change often and we can update them as necessary.

<table>
<thead><tr><th title="Field #1">Directory</th>
<th title="Field #2">Description</th>
<th title="Field #3">Update Schedule</th>
<th title="Field #4">URL</th>
</tr></thead>
<tbody><tr>
<td>annovar</td>
<td>Reference datasets that come with software installation.</td>
<td>Installed when software is installed.</td>
<td>annovar.openbioinformatics.org</td>
</tr>
<tr>
<td>antismash</td>
<td>Reference files and commands for antismash version 4.2.0</td>
<td>Version specific database installed when software is installed</td>
<td>antismash.secondarymetabolites.org</td>
</tr>
<tr>
<td>blast</td>
<td>NCBI nr, nt, refseq_genomic and refseq datasets</td>
<td>Updated on release 6 times a year</td>
<td>www.ncbi.nlm.nih.gov/refseq</td>
</tr>
<tr>
<td>blastv5</td>
<td>Version 5 of NCBI nr, nt, refseq_genomic and refseq datasets. </td>
<td>Updated on release 6 times a year.</td>
<td>www.ncbi.nlm.nih.gov/refseq</td>
</tr>
<tr>
<td>diamond</td>
<td>Diamond reference databases for versions 0.8.38, 0.9.10, 0.9.22 and 0.9.24. Database format periodically changes.</td>
<td>Updated when NCBI nr databases are updated.</td>
<td>ab.inf.uni-tuebingen.de/software/diamond</td>
</tr>
<tr>
<td>gtdbtk</td>
<td> </td>
<td>Version specific database installed when software is installed.</td>
<td> </td>
</tr>
<tr>
<td>gtex</td>
<td>Genotype-Tissue Expression project, comprehensive resource to study tissue-specific gene expression and regulation</td>
<td>Please contact the [Oates lab](mailto:e.oates@unsw.edu.au) for access to a large set of GTEx datathat is not publicly available.</td>
<td>https://gtexportal.org/home/</td>
</tr>
<tr>
<td>hapcol</td>
<td>Reference datasets that come with software installation.</td>
<td>Installed when software is installed.</td>
<td>hapcol.algolab.eu</td>
</tr>
<tr>
<td>hg19</td>
<td>Human reference genome hg19 (GRCh37).</td>
<td>Fixed reference. Never updated.</td>
<td>www.ncbi.nlm.nih.gov/grc</td>
</tr>
<tr>
<td>interproscan</td>
<td>Reference datasets for InterProScan versions 5.20-59.0 and 5.35-74.0</td>
<td>Version specific database installed when software is installed.</td>
<td>www.ebi.ac.uk/interpro</td>
</tr>
<tr>
<td>itasser</td>
<td>Rererence datasets for I-TASSER plus link to current nr database.</td>
<td>Version specific databases installed when software is installed plus link to nr database (see blast above).</td>
<td>zhanglab.ccmb.med.umich.edu/I-TASSER</td>
</tr>
<tr>
<td>kaiju</td>
<td>Reference databases for all versions of Kaiju. Same databases for all versions.</td>
<td>Databases installed when software is installed.</td>
<td>kaiju.binf.ku.dk</td>
</tr>
<tr>
<td>matam</td>
<td>Reference databases for all MATAM versions.</td>
<td>Version specific database installed when software is installed.</td>
<td>github.com/bonsai-team/matam</td>
</tr>
<tr>
<td>megan</td>
<td>Reference databases for all MEGAN versions.</td>
<td>Version specific database installed when software is installed.</td>
<td>ab.inf.uni-tuebingen.de/software/megan6</td>
</tr>
<tr>
<td>repeatmasker</td>
<td>Reference datasets for RepeatMasker version 4.0.7</td>
<td>Version specific database installed when software is installed.</td>
<td>www.repeatmasker.org</td>
</tr>
<tr>
<td>sra</td>
<td>Sequence Read Archive, repository of high throughput sequencing data</td>
<td> </td>
<td>https://www.ncbi.nlm.nih.gov/sra</td>
</tr>
<tr>
<td>trinotate</td>
<td>Reference databases for all versions of Kaiju. Same databases for all versions.</td>
<td>Databases installed when software is installed.</td>
<td>trinotate.github.io</td>
</tr>
</tbody></table>